tests/e2e/mg_utils.py

```diff
@@ -17,6 +17,19 @@ def mg_sleep_and_assert(expected_value, function_to_retrieve_data, max_duration=
     return result
 
 
+def mg_sleep_and_assert_for_a_while(
```

Comment:
Rename this to `mg_assert_after`

Reply:
Changed to `mg_assert_until` as we need to ping for some time